app/cdap/components/DataPrep/DataPrepStatistics/DataPrepInfo/DateTimeInfo.js

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
is there a reason these files are js not tsx?

Reply:
No -- they should've been, fixed!